src/robot_interface/robot_interface.py

```diff
@@ -92,15 +88,11 @@ def log_status(
         raise NotImplementedError
 
     @abstractmethod
-    def get_inspection_references(
-        self, vendor_mission_id: Any, current_task: Task
-    ) -> Sequence[Inspection]:
+    def get_inspection_references(self, current_task: Task) -> Sequence[Inspection]:
```

Comment:
Does it have to be the current task we want to get the inspection reference from? Could this instead be renamed to just task? This is maybe a separate PR; but still an interesting thought.
```suggestion
    def get_inspection_references(self, task: Task) -> Sequence[Inspection]:
```

Reply:
Good idea. It would change the requirements to the implementation of the robot though depending on what the robot does.

For example, would we expect to retrieve inspections for "any" task id? I would assume just for the task IDs in the current mission.

Reply:
Interesting idea. It is now left for #147